client/settings/settings-manager/style.scss

```diff
@@ -0,0 +1,49 @@
+body.woocommerce_page_wc-settings-checkout,
+body.woocommerce_page_wc-settings-checkout .settings-content,
+body.woocommerce_page_wc-settings-checkout #wpbody-content,
+body.woocommerce_page_wc-settings-checkout #wpcontent {
+	background: #fff !important;
+}
+
+body.woocommerce_page_wc-settings-checkout
+	.woocommerce
+	nav.woo-nav-tab-wrapper {
+	display: none !important;
```

Comment:
I personally don't think we should remove the navigation as part of that PR. WDYT? cc @anu-rock.

Reply:
I removed it from here until it will be removed from core. I just added a comment for this.